share/eds/eds.go

```diff
@@ -45,28 +46,35 @@ type writingSession struct {
 // Order: [ Carv1Header | Q1 |  Q2 | Q3 | Q4 | inner nodes ]
 // For more information about the header: https://ipld.io/specs/transport/car/carv1/#header
 func WriteEDS(ctx context.Context, eds *rsmt2d.ExtendedDataSquare, w io.Writer) error {
+	ctx, span := tracer.Start(ctx, "write-eds")
```

Comment:
i suggest that we put span names in constants

Reply:
Applies to the rest of the spans

Reply:
This would hurt readability and is optimized by the compiler to do this anyways